Cards/Movement State/MoveState_InHand.gd

```diff
@@ -2,6 +2,10 @@ extends CardMovementState
 class_name MoveState_InHand
 
 
+@export var lerp_speed: float = 10.0
```

Comment:
should this be renamed to `movement_speed` since functions containing the word `lerp` have been renamed too ?

Reply:
Which functions are you referring to that were renaming? I'm okay with changing the naming if it's more clear

Reply:
for example `get_lerp_component` was renamed to `get_card_movement_component`

Reply:
It’s still technically doing a lerp in the CardMovementCimponent, but I can change it because it’s more clear

Reply:
honestly, I never saw the word lerp before starting this project